source/loader/ze_loader.cpp

```diff
@@ -267,37 +267,43 @@ namespace loader
         auto discoveredDrivers = discoverEnabledDrivers();
         std::string loadLibraryErrorValue;
 
-        std::string home_dir;
+        auto log_directory = getenv_string("ZEL_LOADER_LOG_DIR");
+        if (log_directory.empty()) {
+            std::string home_dir;
 #ifdef _WIN32
-        home_dir = getenv_string("USERPROFILE");
-        if (home_dir == ""){
-            auto home_drive = getenv_string("HOMEDRIVE");
-            auto home_path = getenv_string("HOMEPATH");
-            if ((home_drive != "") && (home_path != "")) {
-                home_dir = home_drive + home_path;
-            } else {
-                home_dir = ".";
+            home_dir = getenv_string("USERPROFILE");
+            if (home_dir == ""){
+                auto home_drive = getenv_string("HOMEDRIVE");
+                auto home_path = getenv_string("HOMEPATH");
+                if ((home_drive != "") && (home_path != "")) {
+                    home_dir = home_drive + home_path;
+                } else {
+                    home_dir = ".";
+                }
```

Comment:
```suggestion
                }
```

This allows `home_dir` to still be empty so line 288 can be true. Line 290 should be what sets it to `.`